pkg/cmd/server/origin/authorizer.go

```diff
@@ -2,73 +2,65 @@ package origin
 
 import (
 	"k8s.io/apiserver/pkg/authentication/user"
-	kauthorizer "k8s.io/apiserver/pkg/authorization/authorizer"
+	"k8s.io/apiserver/pkg/authorization/authorizer"
 	"k8s.io/apiserver/pkg/authorization/authorizerfactory"
 	authorizerunion "k8s.io/apiserver/pkg/authorization/union"
 	"k8s.io/kubernetes/pkg/auth/nodeidentifier"
-	coreinformers "k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/core/internalversion"
 	rbacinformers "k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/rbac/internalversion"
-	rbaclisters "k8s.io/kubernetes/pkg/client/listers/rbac/internalversion"
 	rbacregistryvalidation "k8s.io/kubernetes/pkg/registry/rbac/validation"
 	"k8s.io/kubernetes/plugin/pkg/auth/authorizer/node"
 	rbacauthorizer "k8s.io/kubernetes/plugin/pkg/auth/authorizer/rbac"
 	kbootstrappolicy "k8s.io/kubernetes/plugin/pkg/auth/authorizer/rbac/bootstrappolicy"
 
-	"github.com/openshift/origin/pkg/authorization/authorizer"
+	oauthorizer "github.com/openshift/origin/pkg/authorization/authorizer"
```

Comment:
I think I called this `openshiftauthorizer` elsewhere.  I do not care what we use, but lets make them match.